.changeset/quiet-trainers-study.md

```diff
@@ -0,0 +1,7 @@
+---
+'example-backend': patch
+'@backstage/create-app': patch
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+Bump the gitbeaker dependencies to 28.x
```

Comment:
Should include some quick instructions of how to apply this to an existing app since we update the template.

Reply:
@Rugvip updated!